framework/auth/cas.py

```diff
@@ -160,7 +160,7 @@ def _parse_profile(self, raw, access_token):
 
     def revoke_application_tokens(self, client_id, client_secret):
```

Comment:
Could this method and the one below it be combined to just take a generic payload?
```python
{'client_id': client_id, 'client_secret': client_secret}
``` 
in the case of application tokens and
```python
{'token': token_id}
```
in the case of personal tokens? They're performing the exact same logic just with different ajax payloads.